plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -185,6 +185,7 @@ evaluateScriptRestricting verbose params budget p args = swap $ runWriter @LogOu
 
     let (res, _, logs) =
             UPLC.runCek
+                UPLC.unitCekCosts
```

Comment:
Well this definitely seems wrong! But maybe we can fix it later.

Reply:
Oh yes, that is wrong.  I'll fix that.

Reply:
Fixed.